rskj-core/src/test/java/co/rsk/peg/PegoutCreationEntryTest.java

```diff
@@ -0,0 +1,74 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.crypto.Keccak256;
+import org.apache.commons.lang3.RandomUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotSame;
+
+public class PegoutCreationEntryTest {
+
+    @Test
+    public void testGetPegoutCreationEntryBtcTxHash() {
+        Sha256Hash sha256Hash = PegTestUtils.createHash(RandomUtils.nextInt());
+        Keccak256 keccak256 = PegTestUtils.createHash3(RandomUtils.nextInt());
+        PegoutCreationEntry pegoutCreationEntry = new PegoutCreationEntry(sha256Hash, keccak256);
+        assertEquals(sha256Hash, pegoutCreationEntry.getBtcTxHash());
+        assertNotNull(pegoutCreationEntry.getBtcTxHash());
+    }
+
+    @Test
+    public void testGetPegoutCreationEntryRskTxHash() {
+        Sha256Hash sha256Hash = PegTestUtils.createHash(RandomUtils.nextInt());
+        Keccak256 keccak256 = PegTestUtils.createHash3(RandomUtils.nextInt());
+        PegoutCreationEntry pegoutCreationEntry = new PegoutCreationEntry(sha256Hash, keccak256);
+        assertNotNull(pegoutCreationEntry.getRskTxHash());
+        assertEquals(keccak256, pegoutCreationEntry.getRskTxHash());
+    }
+
+    @Test
+    public void testTestEquals() {
+        Sha256Hash sha256Hash = PegTestUtils.createHash(RandomUtils.nextInt());
+        Keccak256 keccak256 = PegTestUtils.createHash3(RandomUtils.nextInt());
+        PegoutCreationEntry pegoutCreationEntry = new PegoutCreationEntry(sha256Hash, keccak256);
+        PegoutCreationEntry pegoutCreationEntryWithSameValues = new PegoutCreationEntry(sha256Hash, keccak256);
+
+        assertEquals(pegoutCreationEntry, pegoutCreationEntryWithSameValues);
+
+        PegoutCreationEntry pegoutCreationEntryWithDifferentValues = new PegoutCreationEntry(
+            PegTestUtils.createHash(2),
+            PegTestUtils.createHash3(2)
+        );
+
+        assertNotSame(pegoutCreationEntry, pegoutCreationEntryWithDifferentValues);
+    }
+
+    @Test
+    public void testTestHashCode() {
+        Sha256Hash sha256Hash = PegTestUtils.createHash(4);
+        Keccak256 keccak256 = PegTestUtils.createHash3(4);
+        PegoutCreationEntry pegoutCreationEntry = new PegoutCreationEntry(sha256Hash, keccak256);
+        PegoutCreationEntry pegoutCreationEntryWithSameValues = new PegoutCreationEntry(sha256Hash, keccak256);
+
+        assertEquals(pegoutCreationEntry.hashCode(), pegoutCreationEntryWithSameValues.hashCode());
```

Comment:
I think it would be better to test it against the expected value. This way it's not really doing anything, it's like `assertEquals(a, a)`